app/views/suggestions/_suggestion.html.erb

```diff
@@ -0,0 +1,32 @@
+<table>
+	<% @info.each do |pair|%>
+	<% pair_id = pair[0] %>
+	<% food = pair[1] %>
+	<% music = pair[2] %>
+	<tr>
+		<td>
+			<h3>Favorite this pairing!</h3>
+			<% if @favorites.include?(pair_id) == false %>
+				<%=link_to "Favorite This", favorite_path("suggestion" => pair_id), method: :post %>
+			<% else %>
+				One of your faves!
+				<%=link_to "Unfavorite This", destroy_favorite_path("suggestion" => pair_id), method: :delete %>
+			<% end %>
+<!-- 			favorite_path(:pair_id => pair) -->
```

Comment:
Make sure to delete commented out code before your PR (I know you don't have a tool of time, but try to do one last proofread for this one)